src/liger_kernel/ops/layer_norm.py

```diff
@@ -0,0 +1,228 @@
+import math
+import operator
+
+import torch
+import triton
+import triton.language as tl
+
+from liger_kernel.ops.utils import (
+    calculate_settings,
+    compare_version,
+    ensure_contiguous,
+)
+
+if compare_version("triton", operator.ge, "3.0.0"):
+    try:
+        # typical import path with dispatch available
+        from triton.language.extra.libdevice import rsqrt
+    except ModuleNotFoundError:
+        # for working with NGC containers
+        from triton.language.extra.cuda.libdevice import rsqrt
+else:
+    from triton.language.math import rsqrt
+
+
+@triton.jit
+def _layer_norm_forward(
+    Y_ptr,
```

Comment:
looks almost good. Can we add some comments on the shape similar to https://github.com/linkedin/Liger-Kernel/pull/145/files#diff-9778ccd57e6d91be7bf3a5a42d621a010150abcbf55ca5e4be9821b914fea6fcR24?